src/Table/Column/Checkbox.php

```diff
@@ -47,7 +47,7 @@ public function getHeaderCellHtml(Field $field = null, $value = null): string
         }
         $this->table->js(true)->find('.' . $this->class)->checkbox();
 
-        return parent::getHeaderCellHtml($field);
+        return $this->getTag('head', [['div', ['class' => $this->class], [['input/', ['type' => 'checkbox']]]]], []);
```

Comment:
class should be `'ui checkbox ' . $this->class`

Reply:
done